CHANGES.rst

```diff
@@ -12,6 +12,8 @@ Version 7.1
 
 Unreleased
 
+-   Fix dash-started ``args`` after ``--`` from being treated as options in bash completion. (`#1248`_)
```

Comment:
looking at previous lines in this files, yours is much longer, please wrap it similar to the others